src/devices/Device-Index.md

```diff
@@ -65,6 +65,7 @@
 * [Still image recording library](Media/README.md)
 * [TCS3472x Sensors](Tcs3472x/README.md)
 * [TM1637 - Segment Display](Tm1637/README.md)
+* [uFire_ISE - I2C Driver](uFire_ISE/README.md)
```

Comment:
You should not change this file, it is generated automatically. Add a txt file in the root of the directory with the device description